aeron/aeron.go

```diff
@@ -36,10 +36,10 @@ type NewPublicationHandler func(string, int32, int32, int64)
 type NewSubscriptionHandler func(string, int32, int64)
 
 // AvailableImageHandler is the handler type for image available notification from the media driver
-type AvailableImageHandler func(*Image)
+type AvailableImageHandler func(ImageInterface)
```

Comment:
This is a reasonable breaking change. Do we want ImageInterface in the public API? should the concrete data type be renamed instead?